.gitignore

```diff
@@ -14,3 +14,6 @@ raw_reports
 cypress/screenshots
 cypress/videos
 reports
+
+package-lock.json
+package.json
```

Comment:
That won't help your issue, simply don't add these files to commits if there is no need